pages/subscription.tsx

```diff
@@ -13,6 +14,18 @@ export default function ProfilePage() {
 
   let { success, canceled, orgId } = router.query;
 
+  useEffect(() => {
+    if (!router.isReady) return;
+
+    function navigateToDashboardPage() {
+      router.push(Routes.DASHBOARD);
+    }
+
+    if (!parseBooleanEnvValue(window.__env.NEXT_PUBLIC_STRIPE_ENABLED)) {
```

Comment:
Why do you access the window.__env directly instead of using loadConfig which uses it to configure the configuration values?